### DISCUSSION TITLE
Dumping Wrath of the Black Manta gives different results every time

### DISCUSSION THREAD

**Discussion by fakkuyuu** [sanni/cartreader#790] [Need help]
OSCR HW3 V12.4

[+] Add-ons

[+] 80s Consoles

[+] NES/Famicom

Searching database
for 8192DBA3 or 92B115D1...
//////////////When I turn on the power and try to dump for the first time, I always get this CRC//////////////

CRC not found in database
Using manual selection

Printing PRG at 32768
20 b0 d7 a5 00 c9 60 90 03 4c a0 cc 60 a9 00 9d 
79 03 9d 84 03 9d 8f 03 9d 9a 03 ad 00 03 c9 55 
d0 01 60 ad b3 05 f0 06 a9 00 9d 13 04 60 a5 54 
d0 31 a5 52 d0 2d a0 08 b9 13 04 d0 26 8a 48 98 
aa 20 97 96 a9 02 9d 13 04 a9 0d 9d 81 04 a9 50 
9d 8f 03 a9 01 9d 84 03 fe fa 04 20 89 b9 ee a8 
05 68 aa a9 03 8d c6 05 a9 00 9d 8f 03 9d 9a 03 
8a 48 a0 00 98 48 b9 4a c1 d0 03 4c 46 c1 85 0e 
38 e5 54 b0 03 4c 3f c1 c9 10 90 03 4c 3f c1 0a 
0a 0a 0a 85 0f 38 ed 8f 03 85 01 b0 07 a9 00 38 
e5 01 85 01 a5 01 c9 48 90 03 4c 3f c1 c8 b9 4a 
c1 85 10 a2 08 bd 13 04 f0 0a a5 0e dd ce 04 d0 
03 4c 3f c1 e8 e0 0b d0 ec 20 54 d8 f0 78 98 aa 
20 97 96 a9 02 9d 13 04 a9 1c 9d 81 04 a9 50 20 
2e d8 a5 10 9d 79 03 a5 0f 9d 8f 03 a5 0e 9d ce 
04 a9 00 8d b2 05 ee a8 05 ad b0 05 d0 0a a9 17 
85 07 20 7d fe ee b0 05 a5 52 29 0f 85 00 bd 8f 
03 38 e5 00 9d 8f 03 bd 9a 03 e9 00 9d 9a 03 a9 
00 85 00 a5 5e 10 02 c6 00 85 01 18 bd 8f 03 65 
01 9d 8f 03 bd 9a 03 65 00 9d 9a 03 4c 46 c1 68 
a8 c8 c8 4c 74 c0 68 68 aa 60 33 80 2c 80 26 80 
1f 80 1a 80 12 80 00 20 35 d8 60 c1 61 c1 88 c1 
60 20 84 d7 a5 00 c9 30 b0 1d fe 13 04 a9 8f 9d 
3f 04 a9 c1 9d 4a 04 a9 00 9d 55 04 9d 1e 04 9d 
29 04 a9 00 9d 55 04 60 20 e5 96 20 bc 98 60 8d 
08 88 03 08 88 05 0a 88 07 14 88 09 1e 88 09 7f 
8b a5 70 f0 01 60 20 35 d8 b5 c1 89 c2 1f c2 fd 
c1 d0 c1 b6 c1 60 a5 39 29 02 d0 13 ad 13 04 d0 
0e 20 8e 96 20 f3 cb a9 00 9d 08 04 4c f2 c1 60 
20 ee c2 de 08 04 10 12 bd 08 04 29 07 9d 08 04 
de 0b 03 10 05 a9 00 9d 0b 03 de 8f 03 de e4 04 
d0 0a a9 02 9d 13 04 a9 28 9d 58 03 60 20 ee c2 

Printing PRG at 57344
50 60 bd 05 05 85 0e bd 10 05 85 0f a0 00 b1 0e 
30 24 d0 0b a9 e6 85 0e a9 dd 85 0f 4c 0e e0 9d 
58 03 bd 0b 03 9d ce 04 a9 02 9d 13 04 a9 00 9d 
0b 03 c8 4c 8a e0 29 7f f0 20 bd 0b 03 9d ce 04 
a9 04 9d 13 04 a9 01 9d 0b 03 a9 20 9d 58 03 c8 
98 48 20 41 e1 68 a8 4c 8a e0 bd 0b 03 9d ce 04 
a9 03 9d 13 04 a9 02 9d 0b 03 c8 b1 0e 9d 3f 04 
c8 b1 0e 9d 4a 04 c8 a9 00 9d 55 04 9d 1e 04 9d 
29 04 a9 00 9d fd 03 4c 8a e0 98 18 65 0e 9d 05 
05 a9 00 65 0f 9d 10 05 bd 0b 03 dd ce 04 f0 20 
a8 c9 00 d0 0a a8 bd ce 04 c9 01 f0 02 a0 03 98 
9d 97 04 a9 20 1d 60 04 9d 60 04 a9 00 9d 08 04 
60 a0 08 b9 13 04 d0 60 c8 b9 13 04 d0 5a c8 b9 
13 04 d0 54 8a 48 a2 08 a0 00 84 0f 20 97 96 a9 
02 9d 13 04 a9 08 9d 81 04 a9 1e 20 2e d8 a4 0f 
b9 29 e1 9d 08 04 b9 2a e1 18 6d 79 03 9d 79 03 
a9 2f 9d 3f 04 a9 e1 9d 4a 04 a9 00 9d 55 04 9d 
1e 04 9d 29 04 a9 00 9d 55 04 e8 a5 0f 18 69 02 
85 0f c9 02 d0 b6 68 aa 60 04 00 0c e0 14 28 8d 
08 88 03 04 88 05 04 88 07 08 88 09 10 88 0b 7f 
8b a0 08 b9 13 04 f0 08 a0 09 b9 13 04 f0 01 60 
8a 48 98 aa 20 97 96 a9 02 9d 13 04 a9 07 9d 81 
04 68 48 a8 b9 8f 03 18 69 48 9d 8f 03 b9 79 03 
18 69 e8 9d 79 03 a9 1d 20 2e d8 a9 02 9d fd 03 
a9 07 20 ed d6 68 aa 60 ad a2 05 d0 01 60 85 00 
a9 a8 85 01 ac 75 05 f0 23 a9 10 99 00 02 a9 ff 
99 01 02 a9 01 99 02 02 a5 01 99 03 02 18 69 08 
85 01 c8 c8 c8 c8 f0 04 c6 00 d0 dd 8c 75 05 60 
bd c3 04 d0 01 60 c9 08 90 05 a9 08 9d c3 04 de 
c3 04 d0 f1 a9 0d 85 07 20 7d fe 60 a2 07 a9 01 
9d 13 04 a9 00 9d 3f 04 a9 e2 9d 4a 04 a9 08 9d 
55 04 a9 00 9d 08 04 a9 04 9d 58 03 20 ad ec 60 
Set Mapper manually
Set PRG Size
Set CHR Size
Set RAM Size

NES CART READER

CURRENT SETTINGS

MAPPER:   1
PRG SIZE: 128K
CHR SIZE: 128K
RAM SIZE: 0K

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/CART/1495/...
[*******************]
CRC32... 737F8D39 -> Not found

Press Button...
//////////////Reset//////////////
OSCR HW3 V12.4

[+] Add-ons

[+] 80s Consoles

[+] NES/Famicom

Searching database
for 98E94E06 or 92B115D1...

NES CART READER

CURRENT SETTINGS

MAPPER:   1
PRG SIZE: 128K
CHR SIZE: 128K
RAM SIZE: 0K

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1496/...
[*******************]
CRC32... F651398D -> Wrath of the Black Manta (USA).nes

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1497/...
[*******************]
CRC32... 1C7AD672 -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1498/...
[*******************]
CRC32... C6CAB956 -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1499/...
[*******************]
CRC32... 8B22C94D -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1500/...
[*******************]
CRC32... F651398D -> Wrath of the Black Manta (USA).nes

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1501/...
[*******************]
CRC32... 1B892809 -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1502/...
[*******************]
CRC32... CC1A2358 -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1503/...
[*******************]
CRC32... 12BD08AE -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1504/...
[*******************]
CRC32... EF4376CE -> Not found

Press Button...

[+] Read iNES Rom

Saving to NES/ROM/WrathoftheBlack/1505/...
[*******************]
CRC32... E39E67E4 -> Not found

Press Button...

Consecutive dumps like this will give different results each time. Sometimes you can dump successfully.
Checking with ROM checker, PRG CRC seems to be different every time.
![ROM Checker V1 54 2023_05_15 21_15_20](https://github.com/sanni/cartreader/assets/120779915/2557aa26-58e8-4612-90fc-e0eacb421a58)
![16GBSANDISK (F_) 2023_05_15 21_19_52](https://github.com/sanni/cartreader/assets/120779915/46e556da-639f-4a68-ace6-ae44197d945a)
![IMG_20230515_161549](https://github.com/sanni/cartreader/assets/120779915/a6f03f28-9722-4f6d-8b63-0af7a298893b)
![IMG_20230515_161539](https://github.com/sanni/cartreader/assets/120779915/823c8856-866b-4eea-b7df-1a3e82eae1b7)

**Comment by nsx0r** [comment]
![image](https://github.com/sanni/cartreader/assets/13054953/f64a53b2-ec40-4135-a137-330c8e6a1a6c)

**Reply by fakkuyuu** [reply]
no

**Comment by fakkuyuu** [comment]
Similarly to this, there was a MEGA GAMES 10 that was not recognized stably even though the slot and cassette terminals were clean, but when I inserted it just to the left, it was recognized stably and the dump went well.
So blackmanta may also work if you are careful when inserting the adapter or cassette

**Comment by PsyK0p4T** [comment]
if you managed to get the good/expected CRC (and more than once by looking at your logs) then I think you're fine.

As nsx said, it could be due to dirty pins, but it could come from a dirty connector too, or connector with a pin not making a perfect contact with the cart every time.

Also, if you're using a hw v3, I guess you're using a NES cart adapter with it. Some of them which are sold online aren't perfect and use a cartridge slot with a tiny error margin, which makes the connector pins not totally aligned with cartridge pins. I got one like this here. Double-check if you cartridge is perfectly inserted at the very middle of the connector, so the extreme right and left pins are both in contact with the NES slot.

**Comment by splash5** [comment]
Some of adapter sellers sells PCBs with HASL finish (cheaper than ENIG), which makes contacts much easier to corrode.
Choose the adapter that is using ENIG finish (golden finish).

Even the original Famicom / NES has chance to give you gray screen, corrupted graphics (not reading properly),
you should not expect that OSCR will always gives you good dump every time you try.

Just try few more times as @nsx0r said, like what you do on real console.

**Comment by fakkuyuu** [comment]
When I put the NES adapter in the center of the SNES slot, it became stable to dump. I didn't realize there was such a gap left and right. However, the first recognition was still recognized as 8192DBA3 or 92B115D1. I've dumped more than 10 NESs, but only MANTA behaves like this.
thank you everyone.
![IMG_20230517_154459](https://github.com/sanni/cartreader/assets/120779915/eae4724d-2793-4533-896a-2f0ac32479d9)
![IMG_20230517_154507](https://github.com/sanni/cartreader/assets/120779915/7a6fe655-e66c-4ad3-8d00-1ec41581cf90)
![IMG_20230517_154513](https://github.com/sanni/cartreader/assets/120779915/61503d6d-4920-4dcd-b0b4-6ee8f39f11fb)

**Reply by fakkuyuu** [reply]
Also, when the NES cartridge is inserted into the adapter left-aligned, the dump succeeds from the first time, but when it is inserted into the adapter right-aligned, it fails. This may be the real cause.

**Reply by PsyK0p4T** [reply]
Spacing problems can happen between SNES slot and NES adapter pins. But also between the NES slot and NES pcb pins.
It really depends on how the adapter was manufactured, and what component was used for the connector.
In a few words : double check if all the pins are perfectly aligned between cart / slot / adapter.
